runtime/dancebox/Cargo.toml

```diff
@@ -71,6 +71,7 @@ tp-traits = { workspace = true }
 
 # Polkadot
 pallet-xcm = { workspace = true }
+pallet-xcm-benchmarks = { workspace = true }
```

Comment:
pallet-xcm-benchmarks is probably only needed under the runtime-benchmarks feature? can we make it optional in that case?